make/release_flags.mk

```diff
@@ -1,4 +1,5 @@
-VERSION_TAG = $(shell git describe --tags)
+# Create globally-consistent, build-input identifier
```

Comment:
nit: comments should always be full sentences, including punctuation.

Reply:
i.e. "Add a full stop"?
CMIIW but "{Imperative verb} {adjective} {direct objective}" is a complete sentence.

Reply:
yup, just update it to: `# Create globally-consistent, build-input identifier.`

Reply:
i'll do it 👍